VS/VehicleSignals.rdf

```diff
@@ -2964,6 +2967,7 @@
 	
 	<owl:Class rdf:about="&auto-vs;Lidar">
 		<rdfs:subClassOf rdf:resource="&sosa;Sensor"/>
+		<rdfs:label xml:lang="en">Lidar</rdfs:label>
```

Comment:
Note that https://github.com/edmcouncil/fibo/blob/master/ONTOLOGY_GUIDE.md#naming-and-labeling-conventions requires all labels be lower case.